backend/dal/Services/Admin/Concrete/AdministrativeAreaService.cs

```diff
@@ -45,6 +48,57 @@ public AdministrativeArea Get(string name)
                 .FirstOrDefault(c => c.Name == name);
         }
 
+        /// <summary>
+        /// Get the administrative area for the specified id.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        public AdministrativeArea Get(int id)
+        {
+            return this.Context.AdministrativeAreas
+                .AsNoTracking()
```

Comment:
You should generally default to `AsNoTracking()` when making a request for a lot of rows which will only be used as readonly.  When requesting one you probably should default to using tracking.